geomstats/euclidean_space.py

```diff
@@ -19,16 +19,19 @@ def belongs(self, point):
         """
         Check if point belongs to the Euclidean space.
         """
-        point = gs.to_ndarray(point, to_ndim=2)
-        _, point_dim = point.shape
+        point_dim = point.shape[-1]
         return point_dim == self.dimension
 
-    def random_uniform(self, n_samples=1):
+    def random_uniform(self, n_samples=1, n_channels=1):
         """
         Sample a vector uniformly in the Euclidean space,
         with coordinates each between -1. and 1.
         """
-        point = gs.random.rand(n_samples, self.dimension) * 2 - 1
+        point = gs.random.rand(n_samples, n_channels, self.dimension) * 2 - 1
+        if n_channels == 1:
+            point = gs.squeeze(point, axis=1)
+        if n_samples == 1:
```

Comment:
do we want to make it if n_sampes = none? I'm just afraid this is not going to play nice and that some people might expect some [[1.0]] stuff.